invoicing/api/views.py

```diff
@@ -118,7 +118,7 @@ def get(self, request, **kwargs):
         pdf_data = create_invoice_pdf_from_installments(installments)
         apartment = get_apartment(reservation.apartment_uuid)
         title = (apartment.title or "").strip().lower().replace(" ", "_")
-        filename = f"laskut_{title}" if title else "laskut"
+        filename = f"laskut_{title}.pdf" if title else "laskut.pdf"
```

Comment:
👀

Reply:
Adding the file extension so you don't have to select the app to open that file in some stupid OS :D

Reply:
Sure, I was just wondering how that could be missing in the first place 😄